library/eggdrop

```diff
@@ -1,10 +1,14 @@
 Maintainers: Geo Van O <geo@eggheads.org> (@vanosg)
 GitRepo: https://github.com/eggheads/eggdrop-docker.git
 
-Tags: 1.8, 1.8.0, develop
-GitCommit: 49321b7d9b37d7a177a9a1b9c5f14eb13df5e204
+Tags: develop
```

Comment:
You are indeed correct that the develop tag is a periodically updated tag (roughly twice a month is what we’re shooting for). I’m open to other strategies from y’all, but the updates to this project are fairly slow compared to most of the other projects- slow enough (and small enough) that I personally am not overly concerned about any desynch between git and the image- but that is just an opinion. If there is an automated solution to update the image that anyone can clue me in on, that would be a great solution I think as well? Maybe we can chat through some ideas or alternatives in #docker-library tomorrow evening.

 

From: Tianon Gravi [mailto:notifications@github.com] 
Sent: Monday, September 12, 2016 6:55 PM
To: docker-library/official-images <official-images@noreply.github.com>
Cc: Geo <george.vo@alum.rpi.edu>; Author <author@noreply.github.com>
Subject: Re: [docker-library/official-images] Update to Eggdrop 1.8.0rc1 (#2139)

 

In library/eggdrop <https://github.com/docker-library/official-images/pull/2139#discussion_r78476099> :

> @@ -1,10 +1,14 @@
>  Maintainers: Geo Van O <geo@eggheads.org <mailto:geo@eggheads.org> > (@vanosg)
>  GitRepo: https://github.com/eggheads/eggdrop-docker.git
>  
> -Tags: 1.8, 1.8.0, develop
> -GitCommit: 49321b7d9b37d7a177a9a1b9c5f14eb13df5e204
> +Tags: develop

I'm a little bit concerned about the expectations it might create for users of this tag to have it isolated like this (which in turn will give it a dedicated space by itself under "Supported tags" in the image description) -- ie, this isn't going to be always up-to-date with the develop branch from Git (which I'm afraid is the level of support this appears to convey), and is more like a point-in-time snapshot that's likely going to be updated on a semi-regular/sporadic basis. Thoughts?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/docker-library/official-images/pull/2139/files/1fca91fa11d39dde81b46ae69fefd85b030bf860#r78476099> , or mute the thread <https://github.com/notifications/unsubscribe-auth/ACU8sRb64OmANAIEBIOwVKUKCBG66TD6ks5qpeZQgaJpZM4J7HV8> .  <https://github.com/notifications/beacon/ACU8sRjXxYSusk6O3GMhJ8fWcEZ8zj1Lks5qpeZQgaJpZM4J7HV8.gif>